buildlib/dockers/docker-compose-aarch64.yml

```diff
@@ -57,18 +57,6 @@ services:
         CUDA_VERSION: 11.7.0
         NV_DRIVER_VERSION: 515
         ARCH: aarch64
-  centos7-mofed5-cuda12:
```

Comment:
why removed?

Reply:
The base image doesn't support ARM:
https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=12.0.0-devel-centos